src/components/Footer.js

```diff
@@ -1,22 +1,48 @@
 import React from "react";
+import PropTypes from "prop-types";
 
+const Footer = (props) => {
+  const { handleClick, marker } = props;
 
+  return (
+    <footer className="footer">
+      <h1>
+        {" "}
```

Comment:
What does this space do here?

Reply:
I'm thinking this was a "prettier" addition to force a space for visual purposes. I've removed them though and don't see any ill effects, so all good.